### PR TITLE
payment points details

### PR DESCRIPTION
**Pull request by jmiliMP**
Added example for retrieving payment points and updated redirect to github docs on clicking logo.

### CODE REVIEW

**Review by granatoidas — APPROVED** [review]

File: docs/payment-points-api.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Creating, editing payment points is done in _MobilePay Portal_. The payment poin
 
 ## Retrieve a list of payment points
 
-The following request retrieves information about all the payment points for a merchant account usable with Payments API.
+In order to get the details of payment points like `PaymentPointId`, you need to call Payment Points Api. The following request retrieves information about all the payment points for a merchant account usable with Payments product.
```
Comment (edvinasmuncelis) [inline docs/payment-points-api.md:23]: API should be all uppercase letters since it's an acronym.

Comment (edvinasmuncelis) [reply docs/payment-points-api.md:23]: "_you need to use Payment Points API_"

Comment (edvinasmuncelis) [reply docs/payment-points-api.md:23]: remove "the"

Comment (edvinasmuncelis) [reply docs/payment-points-api.md:23]: replace "product" with "API"

**Review by edvinasmuncelis — CHANGES_REQUESTED** [review]

**Review by edvinasmuncelis — APPROVED** [review]

**Pull request merged**